static/js/components/CommentForms.js

```diff
@@ -72,6 +72,14 @@ const getPostReplyInitialValue = (parent: Post) => ({
   text:    ""
 })
 
+const userConditionalFunction = (
```

Comment:
can we do a more description name for this? just something that captures the internal logic a little better